autogen.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 
-if glibtoolize --version &> /dev/null
+glibtoolize=$(which glibtoolize)
```

Comment:
jep, is working, but I don't see the point. 
If `which` doesn't find the binary in PATH, it shows no output